### PR TITLE
Update kubernetes-blue-deployment.yml

### PR DESCRIPTION
**Pull request by RichardZhaoW**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: cli/endpoints/online/kubernetes/kubernetes-blue-deployment.yml [hunk 1]
```diff
@@ -25,13 +25,13 @@ liveness_probe:
   period: 5
   timeout: 10
```
Comment (mjaow) [inline cli/endpoints/online/kubernetes/kubernetes-blue-deployment.yml:26]: about all timeout = 120, align with v1

File: cli/endpoints/online/kubernetes/kubernetes-blue-deployment.yml [hunk 2]
```diff
@@ -25,13 +25,13 @@ liveness_probe:
...
   success_threshold: 1
-  failure_threshold: 1
+  failure_threshold: 5
 readiness_probe:
   initial_delay: 5
   period: 5
   timeout: 10
```
Comment (mjaow) [inline cli/endpoints/online/kubernetes/kubernetes-blue-deployment.yml:32]: about all timeout = 120, align with v1

**Review by snowei — APPROVED** [review]

**Review by mjaow — APPROVED** [review]

File: cli/endpoints/online/kubernetes/kubernetes-blue-deployment.yml (hunk 1 above)
Comment (RichardZhaoW) [reply cli/endpoints/online/kubernetes/kubernetes-blue-deployment.yml]: V2 has different default values with V1.

**Review by zetiaatgithub — APPROVED** [review]

**Comment by RichardZhaoW** [comment]
update the default values for the v2 inference runner.

**Pull request merged**